sechub-shared-kernel/src/test/java/com/daimler/sechub/sharedkernel/resilience/ResilientActionExecutorTest.java

```diff
@@ -98,6 +98,58 @@ public void no_error_but_consultant_defined_just_runs_through_and_returns_result
 
 	}
 
+	@Test
+    public void retry_3_times_allowed__we_three_errors_which_forces_a_retry_the_callback_is_called_3_times()throws Exception {
```

Comment:
Is the `we_three_errors` a spelling mistake?

Reply:
Yipp. Typo... will be fixed.